### PR TITLE
Fixes #15553: Create the plugin base

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15553

### CODE REVIEW

File: cis/techniques/cis/category.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<xml>
+    <name>CIS techniques set</name>
+    <description>
+        Define techniques to help audit the CIS RedHat Benchmark with Rudder.
```
Comment (amousset) [inline cis/techniques/cis/category.xml]: not particularly linked to Red Hat

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

File: cis/ncf/30_generic_methods/cis/partition_check_mounted.cf [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      "class_prefix"           string => string_head("${full_class_prefix}", "1000");
+
+  classes:
+      "pass3"              expression => "pass2";
+      "pass2"              expression => "pass1";
+      "pass1"              expression => "any";
+
+      "findmnt_found"      expression => isexecutable("/bin/findmnt");
+    findmnt_found.pass2.!pass3::
+      "${canonified_mount_point}" expression => returnszero("/bin/findmnt -M ${mount_point}", "useshell");
```
Comment (Fdall) [inline cis/ncf/30_generic_methods/cis/partition_check_mounted.cf]: This option does not exist on the findmnt on centos7

File: cis/ncf/30_generic_methods/cis/partition_check_options.cf [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      "canonified_mount_point" string => canonify("${mount_point}");
+      "old_class_prefix"       string => canonify("partition_check_options_${mount_point}");
+      "args"                    slist => { "${mount_point}", "${options}" };
+      "report_param"           string => join("_", args);
+      "full_class_prefix"      string => canonify("partition_check_options_${report_param}");
+      "class_prefix"           string => string_head("${full_class_prefix}", "1000");
+
+
+    mounted::
+      "json_data" data => parsejson(execresult("/bin/findmnt -JM ${mount_point}", "useshell"));
```
Comment (Fdall) [inline cis/ncf/30_generic_methods/cis/partition_check_options.cf]: These options do not exist on the findmnt on centos7

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**